src/services/document-service.js

```diff
@@ -82,16 +84,20 @@ class DocumentService {
     this.userService = userService;
   }
 
-  getAllDocumentsMetadata() {
-    return this.documentStore.find({}, { sort: lastUpdatedFirst, projection: metadataProjection });
-  }
-
-  getDocumentsMetadataByKeys(documentKeys) {
-    return this.documentStore.find({ _id: { $in: documentKeys } }, { sort: lastUpdatedFirst, projection: metadataProjection });
+  getAllDocumentsMetadata({ includeArchived } = {}) {
+    const filter = {};
+    if (!includeArchived) {
+      filter.archived = false;
+    }
+    return this.documentStore.find(filter, { sort: lastUpdatedFirst, projection: metadataProjection });
   }
 
-  async getDocumentsByTags(searchTags) {
-    const result = await this.documentStore.find({ tags: { $all: searchTags } }, { projection: searchResultsProjection });
+  async getDocumentsByTags(tags, { includeArchived } = {}) {
```

Comment:
Why not always include the tags, I guess those should be relevant anyway if they were included in the system at some point

Reply:
Um, what do you mean? I changed nothing regarding tags, apart from streamlining their name.

Reply:
Ahm, my bad, I didn't read which method this was.